### DISCUSSION TITLE
Is there any way to make the admin panel rtl?

### DISCUSSION THREAD

**Discussion by mahdirezaei-dev** [shopperlabs/framework#256] [Q&A]
I want to use shopper for a straight language, is there a way?

![Untitled](https://github.com/shopperlabs/shopper/assets/53124341/4cec118c-366f-42ad-9c9e-ab9a1aa3a0cb)

**Comment by mckenziearts** [comment]
Hey @mahdirezaei-dev, RTL is a fair request and it's been on our radar for a while.

It isn't shipped today, but we'd happily accept a community PR that introduces an `app-rtl.css` build alongside the main stylesheet and switches it based on the active locale direction. Most of the layout uses Tailwind's logical properties already, so the lift is smaller than it looks.

If you'd like to tackle it, open a PR and we'll help you shape the approach. If not, a thumbs-up on this thread helps us gauge demand for prioritization.
